.changelog/1815.bugfix.md

```diff
@@ -0,0 +1 @@
+i18n: Update translations from Transifex
```

Comment:
```suggestion
i18n: Update translations from Transifex and add normalization script

The translation procedure was slightly changed. On Transifex, use "Download
file to translate" instead of "Download for use". This will download the version
of the file which includes empty strings for missing translations. Remove empty
strings by calling `yarn extract-messages` which invokes the new
`normalize-translations` script. On the UI, React will replace them with the
English versions automatically.
```